README.md

```diff
@@ -89,6 +89,22 @@ XFC.Consumer.mount(document.body, 'http://localprovider.com:8080/example/provide
 
 Here `iframeAttrs` is an object that contains entries, each of them being an entry of attribute's name and value.
 
+### Applying Visual Focus Indicator on Iframe
+Certain configuration within this library such as `fixedWidth` or `fixedHeight` may prevent the content to be fully displayed within the viewport, and thereby needing `scrolling` to be enabled. For keyboard only users, it is important to display a visual focus indicator to indicate where the focus currently is at to help guide them with page navigation.
+
+To allow the styles to be applied for `focus` and `blur` events, please provide the CSS styles for each in the `focusIndicator` object.
```

Comment:
Am I right to say that `focusStyleStr` and `blurStyleStr` are assumed to be inverses of each other? With how this is set up, I'm thinking that if every property in `focusStyleStr` is not also in `blurStyleStr`, then they may not be removed from the frame. Could that could be documented here?

Reply:
Added a note on it below.